ansible_collections/arista/avd/tests/integration/targets/yaml_templates_to_facts/tests/test_with_debug.yml

```diff
@@ -0,0 +1,29 @@
+- name: Test with debug
+  ignore_errors: no
+  vars:
+    timezone: "test"
+    hours: 36
+  register: result
+  yaml_templates_to_facts:
+    debug: true
+    templates:
+      - template: "templates/template.j2"
+
+- assert:
+    that:
+      - result is success
+      - result.changed == true
+      - result.ansible_facts is defined
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug is defined
+      # avd_yaml_templates_to_facts_debug could have been set by a previous test, so we accept a longer list and test only on the last element
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug | length >= 1
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug[-1].timestamps is defined
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug[-1].template is defined
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug[-1].template == "templates/template.j2"
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug[-1].timestamps.starting is defined
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug[-1].timestamps.run_template is defined
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug[-1].timestamps.load_yaml is defined
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug[-1].timestamps.strip_empty_keys is defined
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug[-1].timestamps.combine_data is defined
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug[-1].timestamps.done is defined
+      - result.ansible_facts.avd_yaml_templates_to_facts_debug[-1].timestamps.run_template is defined
```

Comment:
Looks like duplicate assert statement added by mistakenly. I think there is no need of last assert statement.

Reply:
I will fix. Thanks